hbase-client/src/main/java/org/apache/hadoop/hbase/client/RpcConnectionRegistryURIFactory.java

```diff
@@ -19,6 +19,7 @@
 
 import java.io.IOException;
 import java.net.URI;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Sorry, I forget which dependencies we expose transitively. Should we be using a shaded version of this class?

Reply:
Thanks.

Reply:
We do not shade commons-lang3 in hbase-thirdparty

Reply:
Okay thanks for checking. I wonder if we should.

Reply:
Commons-lang3 is not likely to introduce big conflicts. It changed its package name to commons-lang3 from commons-lang when introducing breaking changes.